common/nym-lp/src/keypair.rs

```diff
@@ -0,0 +1,200 @@
+use std::fmt::{self, Display, Formatter};
+use std::ops::Deref;
+use std::str::FromStr;
+
+use nym_sphinx::{PrivateKey as SphinxPrivateKey, PublicKey as SphinxPublicKey};
+use serde::Serialize;
+use utoipa::ToSchema;
+
+use crate::LpError;
+
+#[derive(Clone)]
+pub struct PrivateKey(SphinxPrivateKey);
```

Comment:
do we want to be piggy backing on the Sphinx key here? why should LP be concerned with sphinx' key rotation for example?